api/src/models/user.ts

```diff
@@ -38,6 +38,7 @@ const User = new mongoose.Schema({
   submittedPitches: [{ type: Schema.Types.ObjectId, ref: 'Pitch' }],
   teams: [{ type: Schema.Types.ObjectId, ref: 'Team' }],
   involvementResponse: { type: String, default: null },
+  rejectReasoning: { type: String, default: null },
```

Comment:
I realized that they could still fill out reasoning even if they approve, so could this get renamed to onboardReasoning?